plugins/woocommerce/src/Internal/Admin/ProductReviews/Reviews.php

```diff
@@ -537,7 +537,7 @@ protected function edit_comments_screen_text( $translation, $text ) {
 
 		if ( isset( $comment->comment_parent ) && $comment->comment_parent > 0 ) {
 			$is_reply = true;
-			$comment  = get_comment( $comment->comment_parent ); // phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
+			$comment  = get_comment($comment_id); // phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
```

Comment:
I don't think this line is really needed at all. Removing it should have the same effect (of fixing the issue).
If `$comment` is already set (which it is, as this inside an `if` branch that tests for `$comment->comment_parent`), then it shouldn't be necessary to re-load the whole `$comment` again. That should've happened on line 533 above.